### DISCUSSION TITLE
Crazyflie flight stability

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#975] [Q&A]
Hi, I am recently trying to use uwb for indoor navigation. There is often a situation where the first flight is very successful, but using the same program code, the crazyflie will start to become unstable the second time, or even fly to a coordinate position that is not set, causing loss of control. Does anyone can tell what is led for this situation ?
thanks

**Comment by knmcguire** [comment]
Hi! 
I'm assuming now that you went through this whole process: https://www.bitcraze.io/documentation/tutorials/getting-started-with-loco-positioning-system/

Have you tried looking at the state estimate of the x,y,z position before taking off?  Perhaps just try to take off with the f[light control tab]( https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/flightcontrol_tab/ ). It might be that you are taking off before the position had time to converge. 

Also, with UWB you need to start up the crazyflie facing 1 direction, namely the positive x-axis.

**Reply by CalvinKoung** [reply]
ok, thanks
